Lab6

yinyangcolby opened the pull request:
Still working on EF core and databases from last week before I finish this one up. The goal is to get this all caught up by Friday, the due date that prof Bird pushed last weeks' lab to.

yinyangcolby committed:
Lab 4

Lab 4 submission

yinyangcolby committed:
Lab 3 Updated So Forms Work

Forums works now.

yinyangcolby committed:
Pages updated with correct navbar

yinyangcolby committed:
Forums has bootstrap

yinyangcolby committed:
Unit testing and quiz added

yinyangcolby committed:
Updated xUnit version

xUnit version 2.4.1 was installed, causing an error in the tests. This issue is fixed and now all tests are working.

readyR0840 commented:
Ok, that makes sense, yet I can still do a review on your repo test stuff before then. I can come back tomorrow, and do it then? I think this lab is more about unit tests and controllers for a fake repo, so it wouldn't make sense to review anything, I think. I will ask Brian tomorrow what to do if he is doing a class tomorrow. I think he might let me turn in a late one, but probably before Friday's due date.

yinyangcolby committed:
EFCore and Database

Nuget packages added, forums has been refactored to include database. You can leave feedback and it will remain there.

readyR0840 commented:
Does it compile and run without errors 				              	 	       [N/a ]   
                                                                     
Do all the pages load correctly?							                                  [ N/a]   
                                                                     
Do all the links, buttons or other UI elements work correctly?  	     [ N/a]   
                                                                     
Does the web app provide a good user experience (UX)?       	        [ N/a]   
                                                                     
Does the naming and structure conform to MVC conventions?	     [Looks good]
<br />(Names of controllers, controller methods, views, etc.)   
                                                                     
Does the style conform to C# coding conventions?              		         [ Yes]   
                                                                     
Do the design and implementation conform to best practices?  	     [Yes ]   
                                                                     
Are there appropriate unit tests and do they all pass?                            [No]


Does the solution meet all the requirements in the lab instructions?    [No]   
                                                                     
Additional comments:  As you already know, the lab was not complete, but I thought I would fill this out anyway. I don't think it would be worth downloading this version, but nice job with getting the database going.

readyR0840 reviewed (commented):
Does it compile and run without errors [N/a ]

Do all the pages load correctly? [ N/a]

Do all the links, buttons or other UI elements work correctly? [ N/a]

Does the web app provide a good user experience (UX)? [ N/a]

Does the naming and structure conform to MVC conventions? [Looks good]

(Names of controllers, controller methods, views, etc.)

Does the style conform to C# coding conventions? [ Yes]

Do the design and implementation conform to best practices? [Yes ]

Are there appropriate unit tests and do they all pass? [No]

Does the solution meet all the requirements in the lab instructions? [No]

Additional comments: As you already know, the lab was not complete, but I thought I would fill this out anyway. I don't think it would be worth downloading this version, but nice job with getting the database going.
